lib/pugin/view_helpers.rb

```diff
@@ -8,4 +8,52 @@ def navigation_link(route_symbol:, letter:, primary_id: nil, secondary_id: nil)
 			return link_to(letter, url, data: { 'atoz-see': 'true' })
 		end
 	end
+	module Feature
+		module Status_banner
+			class << self
+				def enable
+				  true
+				end
+				def disable
+					false
+				end
+			end
+		end
+		module Top_navigation
+			class << self
+				def enable
+					true
+				end
+				def disable
+					false
+				end
+			end
+		end
+		module Bandiera
+			class << self
+				def dissolution?
+					get_features
+
+					@features['show-dissolution']
```

Comment:
try `@features.fetch('show-dissolution', false)` as this will default to be explicitly a false value rather than `nil` if the feature is not found